core/pom.xml

```diff
@@ -132,6 +132,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-email</artifactId>
         </dependency>
+		<dependency>
```

Comment:
I dont think we should depend on ACS Commmons... i guess this should really work with any Sling-based error handling?